src/main/java/org/openrewrite/java/spring/boot2/ConvertToSecurityDslVisitor.java

```diff
@@ -203,9 +203,14 @@ private List<J.MethodInvocation> computeAndMarkChain() {
                         .withName(initialMethodInvocation.getName().withSimpleName(methodType.getName()))));
         cursor = cursor.getParent(2);
         if (isApplicableCallCursor(cursor)) {
-            for (; cursor != null && isApplicableCallCursor(cursor); cursor = cursor.getParent(2)) {
```

Comment:
can we add a test case like `.csrf().ignoringRequestMatchers("").disable()` ?
the config before disabling is ineffective, but not unrealistic for code in the wild, I think. and since this condition to catch `disabled` isn't interacting with the cursor-moving loop at all, now, I'm suspicious that this case might not get handled cleanly.

Reply:
Added via https://github.com/openrewrite/rewrite-spring/commit/7bbd005639796a227f4765b0c4e1cd755fefa44a.